CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
   internet access checks. #1402
 - Disused traceroute binaries. #1397
 - "Back door user" post-breach action. #1410
+- Stale code. #1431
```

Comment:
In general, we should prefer the issue number to the PR number. The issues generally give more context than the PRs, and you can make your way to the PR from the issue.

This entry needs a little more detail for a human to understand what this change is.
```suggestion
- Stale code in the Windows system info collector that collected installed packages and WMI info. #1431
```